test/test_constructors.jl

```diff
@@ -62,6 +62,42 @@
                 @test Structure(fbr) == Structure(Tensor(Lvl{Int16}(Element(0.0))))
                 @test Structure(fbr) == Structure(Tensor(Lvl(Element(0.0), Int16(0))))
                 @test Structure(fbr) == Structure(Tensor(Lvl{Int16}(Element(0.0), 0)))
+
+                if key == "SingleList" || key == "SingleRLE"
```

Comment:
These are failing because you have changed the fill so you are trying to write a bunch of non-fill elements (all zero) to the level, but singlelist only supports one non-fill element.

Reply:
Perhaps let's just test everything but the copyto call for these and call it a a day haha. we can just move these a few blocks down.

Reply:
I wanted to move the `if` statement a few blocks below but I noticed that `SingleList` complains when placed in a >=2-dim tensor: https://github.com/willow-ahrens/Finch.jl/issues/447

Reply:
Thank you! Now it works as expected.